### DISCUSSION TITLE
Taking a snapshot from PDF

### DISCUSSION THREAD

**Discussion by johnfrits** [dlemstra/Magick.NET#641] [Help] [answered]
Hi, do you support taking a snapshot of PDF and converting it to Image?

**Comment by dlemstra** [comment]
Can you clarify what you mean by a snapshot of a PDF?

**Reply by johnfrits** [reply]
@dlemstra  I have found it already in one of your docs. I was looking for a library that take a snapshot on any page of PDF and convert to image ( .png )

**Comment by johnfrits** [comment, marked as answer]
For those people who will bump on this link and looking for answers, I have made an example how to do it. 

You should install the ImageMagick the latest version, and if you are doing it on C# you can test this running code and try it. 

https://dotnetfiddle.net/OT0UHo
